tests/nat/runtime/test_runner_trace_ids.py

```
@@ -95,3 +104,71 @@ async def test_runner_trace_and_run_ids(existing_trace: bool, existing_run: bool
+        name = config_name
+        type = config_type
+
+    class _TestFunction:
+        has_single_output = True
+        has_streaming_output = False
+        config = _TestConfig()
+
+        def __init__(self):
+            self.instance_name = instance_name
+
+        def convert(self, v, to_type):
```

Ruff (0.14.14)

Unused method argument: `to_type`

(ARG002)